searxng/templates/configmap.yaml

```diff
@@ -1,11 +1,9 @@
-{{- include "common.values.setup" . -}}
----
 apiVersion: v1
-kind: Secret
+kind: ConfigMap
```

Comment:
The searx config may include some credentials so I don't understand why you changed that to a configmap.